kapitan/inventory/model/input_types.py

```diff
@@ -39,7 +39,7 @@ class KapitanInputTypeBaseConfig(BaseModel):
 class KapitanInputTypeJsonnetConfig(KapitanInputTypeBaseConfig):
     input_type: Literal[InputTypes.JSONNET] = InputTypes.JSONNET
     input_paths: List[str]
-    prune: bool = False
+    prune: Optional[bool] = False
```

Comment:
Hey @alledm.

What do you think of making this default to False in the baseconfig on line 36?

I also noticed that configuring this in the inventory (`prune: false`) config does not work

```
parameters:
  kapitan:
    compile:

    - output_path: manifests
      input_type: kadet
      output_type: yaml
      prune: false
```

Reply:
Yes makes sense. this is something that didn't notice. will fix it soon

Reply:
Ideally this should be default `None` as the setting precedence seems to be:
1. This class' `prune` attribute. 
2. The `prune` cmd/dot setting.
3. Default `False`.

Although while this key exists in the class at all, neither the cmd nor the dot files will do anything. This is because whenever we check if the `prune` key exists rather than if an attempt to get it's value returns `None` at the relevant call sites.